src/libexpr/eval.hh

```diff
@@ -709,10 +709,26 @@ public:
     void concatLists(Value & v, size_t nrLists, Value * * lists, const PosIdx pos, std::string_view errorCtx);
 
     /**
-     * Print statistics.
+     * Print statistics, if enabled.
+     *
+     * Performs a full memory GC before printing the statistics, so that the
+     * GC statistics are more accurate.
      */
     void printStats();
 
+    /**
+     * Print statistics, unconditionally, cheaply, without performing a GC first.
+     */
+    void printStatistics();
```

Comment:
```suggestion
    void printStat();
```

Reply:
Would prefer not to commit this suggestion, because it would silently change the meaning of the method between releases, as it collides the new, smaller helper method with the old name. By leaving it as `printStatistics`, we require human intervention, providing an opportunity for library consumers to pick the right method.